crrExistingObjects.js

```diff
@@ -149,26 +149,9 @@ function _markObjectPending(
             if (skip) {
                 return next();
             }
-            const { Rules, Role } = repConfig;
-            const destination = Rules[0].Destination.Bucket;
-            // set replication properties
-            const ops = objMD.getContentLength() === 0 ? ['METADATA']
-                : ['METADATA', 'DATA'];
-            const backends = [{
-                site: storageClass,
-                status: 'PENDING',
-                dataStoreVersionId: '',
-            }];
-            const replicationInfo = {
-                status: 'PENDING',
-                backends,
-                content: ops,
-                destination,
-                storageClass,
-                role: Role,
-                storageType: STORAGE_TYPE,
-            };
-            objMD.setReplicationInfo(replicationInfo);
+
+            objMD.setReplicationSiteStatus(storageClass, TARGET_REPLICATION_STATUS);
```

Comment:
I think we want to set `'PENDING'` here, not `TARGET_REPLICATION_STATUS`: the target is what objects are targeted for the update, not what status we want to put in metadata.

Also, I don't know if that would work in all cases: for example, new objects don't have any replication status in their metadata, and it's an important use case of the script to be able to replicate new objects without any replication config attached to their bucket when they were created.